### PR TITLE
Ogr-source-scheduling

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**

### CODE REVIEW

**Comment by ChristianBeilschmidt** [comment]
[OGR-Source: Nur für einen Poll blockieren](https://trello.com/c/v1fDsAFg/505-ogr-source-nur-f%C3%BCr-einen-poll-blockieren)

**Review by jdroenner — APPROVED** [review]

File: datatypes/Cargo.toml [hunk 1]
```diff
@@ -17,7 +17,7 @@ pro = ["postgres"]
 arrow = { version = "6.0", features = ["simd"] }
 chrono = "0.4"
 float-cmp = "0.9"
-gdal  = "^0.11"
+gdal = { version = "0.11", features = ["bindgen"] }
```
Comment (jdroenner) [inline datatypes/Cargo.toml:20]: please remove the bindgen feature

File: operators/Cargo.toml [hunk 2]
```diff
@@ -20,7 +20,8 @@ crossbeam = "0.8"
 csv = "1.1"
 float-cmp = "0.9"
 futures = "0.3"
-gdal = "^0.11"
+gdal = { version = "0.11", features = ["bindgen"] }
```
Comment (jdroenner) [inline operators/Cargo.toml:23]: please remove the bindgen feature

**Review by jdroenner — CHANGES_REQUESTED** [review]

File: operators/Cargo.toml (hunk 2 above)
Comment (ChristianBeilschmidt) [reply operators/Cargo.toml]: 23018a9

File: datatypes/Cargo.toml (hunk 1 above)
Comment (ChristianBeilschmidt) [reply datatypes/Cargo.toml]: 23018a9

**Review by jdroenner — APPROVED** [review]

**Comment by jdroenner** [comment]
bors merge

**Pull request merged**